nuxeo-drive-client/nxdrive/manager.py

```diff
@@ -6,6 +6,10 @@
 import sys
 import logging
 from urlparse import urlparse
+import urllib2
+import js2py.pyjs
```

Comment:
You could also use the setup.py includes array, it would be better to avoid later cleanup for python p8p. What do you think ?